test/Calculations/test_Count.py

```diff
@@ -0,0 +1,30 @@
+from src.Calculations.Count import Count, get_calculation
+
+def test_get_calculation():
+    '''
+    Make sure the get_calculation function returns the calculation
+    '''
+    assert get_calculation() == Count
+
+def test_sum():
+    '''
+    Test functionality of the Sum class
```

Comment:
fix this comment to apply to this function instead of Sum (applies to all tests)

Reply:
everything else looks good, just still need to change the comment to say Count class instead of Sum class (applies to rest of tests as well)